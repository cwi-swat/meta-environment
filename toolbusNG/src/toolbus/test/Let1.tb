process P is
printf("top\n") .
let X : int in X := 10 . printf("X = %d\n", X) endlet .
let X : int in X := 20  . printf("X = %d\n", X) endlet .
let X : int in 
    X := 100 .
    printf("X = %d\n", X) .
    let X : int in X := 200 . printf("X = %d\n", X) endlet .
    printf("X = %d\n", X)
endlet .
printf("top\n")

toolbus(P)
