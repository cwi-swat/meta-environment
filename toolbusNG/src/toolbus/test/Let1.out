top
X = 10
X = 20
X = 100
X = 200
X = 100
top
